scripts/format_txts.py

```
@@ -1,28 +1,60 @@
+    from constants import CONTACT_STORE_FILE, DEFAULT_TIMEZONE, OUTPUT_DIR, TMP_PATH
+    from utils import (
+        clean_message_content,
+        is_date_in_future,
+        merge_text_files,
+        parse_date_string,
+        read_json_file,
+        read_text_file,
+        write_json_file,
+    )
+
+script_dir = Path(os.path.dirname(os.path.abspath(__file__)))
```

Ruff (0.8.2)

`os.path.dirname()` should be replaced by `Path.parent`

(PTH120)